### PR TITLE
Rails flavored ruby

### PR DESCRIPTION
**Pull request by ngovannghiaframgia**
ready

### CODE REVIEW

**Comment by ngovannghiaframgia** [comment]
ready

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (huongnguyenmta) [inline app/assets/javascripts/static_pages.coffee]: remove this file if not use

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/static_pages.scss]: remove this file

File: app/controllers/static_pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
```
Comment (huongnguyenmta) [inline app/controllers/static_pages_controller.rb]: => `def home; end`

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
-end
+
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
```
Comment (huongnguyenmta) [inline app/helpers/application_helper.rb:4]: remove ()

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,12 @@
...
+  	base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (huongnguyenmta) [inline app/helpers/application_helper.rb:5]: I18n

File: app/helpers/application_helper.rb [hunk 6]
```diff
@@ -1,2 +1,12 @@
...
+  	if page_title.empty?
+  		base_title
+  	else
+  		page_title + " | " + base_title
+  	end
+  end
+end
```
Comment (huongnguyenmta) [inline app/helpers/application_helper.rb]: end file

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Chuong03</title>
+    <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all',
```
Comment (huongnguyenmta) [inline app/views/layouts/application.html.erb]: replace single quote => double quote

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,15 +1,13 @@
...
+                                              'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
     <%= yield %>
   </body>
-</html>
+</html>
```
Comment (huongnguyenmta) [inline app/views/layouts/application.html.erb]: end file

File: app/views/static_pages/about.html.erb [hunk 9]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
```
Comment (huongnguyenmta) [inline app/views/static_pages/about.html.erb]: I18n

File: app/views/static_pages/about.html.erb [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a> is a
+  <a href="http://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+  This is the sample application for the tutorial.
+</p>
```
Comment (huongnguyenmta) [inline app/views/static_pages/about.html.erb]: end file

File: app/views/static_pages/help.html.erb [hunk 11]
```diff
@@ -0,0 +1,9 @@
+<% provide(:title, "Help") %>
```
Comment (huongnguyenmta) [inline app/views/static_pages/help.html.erb]: the same

File: app/views/static_pages/home.html.erb [hunk 12]
```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Home") %>
```
Comment (huongnguyenmta) [inline app/views/static_pages/home.html.erb]: the same

**Comment by huongnguyenmta** [comment]
commented

**Comment by ngovannghiaframgia** [comment]
ready

**Comment by hungnh103** [comment]
commented

**Pull request merged**